templates/WebApplicationBasic/WebApplicationBasic.csproj

```diff
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <PreserveCompilationContext>true</PreserveCompilationContext>
+    <AssemblyName>WebApplicationBasic</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <PackageId>WebApplicationBasic</PackageId>
+    <RuntimeFrameworkVersion>1.1.0</RuntimeFrameworkVersion>
+    <PackageTargetFallback>$(PackageTargetFallback);dotnet5.6;portable-net45+win8</PackageTargetFallback>
```

Comment:
Can these be dropped? I didn't see anything in the list of dependencies that would require these fallback frameworks anymore.